src/topp/CometAdapter.cpp

```diff
@@ -751,6 +754,15 @@ class TOPPCometAdapter :
     // if "reindex" parameter is set to true will perform reindexing
     if (auto ret = reindex_(protein_identifications, peptide_identifications); ret != EXECUTION_OK) return ret;
 
+	// Parse IM information if present
+	if (exp.hasIM())
```

Comment:
@timosachsenberg @jpfeuffer 
Any preference on how to check if IM is present?

Could have a new `hasIM` function in `MSExperiment` or just use `exp.getFirstProductSpectrum(0).getDriftTime != 0.0` to check

Reply:
@cbielow